jdbc_bridge/src/main/java/com/starrocks/jdbcbridge/JDBCUtil.java

```diff
@@ -0,0 +1,14 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.jdbcbridge;
+
+import java.sql.Date;
+import java.text.SimpleDateFormat;
+
+public class JDBCUtil {
+    // format Date to 'YYYY-MM-dd'
+    static String formatDate(Date date) {
+        SimpleDateFormat format = new SimpleDateFormat("yyyy-MM-dd");
```

Comment:
Maybe better to make the formater as a static member instead of creating every time

Reply:
> Maybe better to make the formater as a static member instead of creating every time

fixed, thanks for your advice